assets/collage.css

```diff
@@ -183,3 +183,69 @@
 .collage .collage-card-spacing img {
   object-fit: contain;
 }
+
+.collage--collapsed-y .card-wrapper,
+.collage--collapsed-y .collage-card {
+  height: calc(100% + var(--card-border-width));
+}
+.collage--collapsed-x .collage-card {
+  width: calc(100% + var(--card-border-width));
+}
```

Comment:
Good question. I think this is here to override the `.collage-item > * style { width: 100% }` at the top of collage.css. It's possible I could be reseting to `width: auto` instead, but this probably seemed a little more explicit at the time. The difference I'm sure has to do with where the actual border styles are being applied relative to the where any width/height dimensions are being applied. The height: 100% calc stuff is needed for both because we want cards to stretch to fill the row height (while still using border-box sizing).